.travis.yml

```diff
@@ -1,146 +1,160 @@
-dist: xenial
```

Comment:
Changes in this file does 2 things:
1. Reverts to the point before this [PR](https://github.com/eventespresso/event-espresso-core/pull/2292)
2. And adds `yarn: true` and `- $HOME/.yarn-cache` options to `cache`